locales/fr_FR/pcsx2_Main.po

```diff
@@ -70,7 +70,7 @@ msgid ""
 "account rights."
 msgstr ""
 "Vous ne disposez pas des droits suffisants pour ouvrir ce fichier (mauvais "
-"compte d'utilisateur?)."
+"compte d'utilisateur ?)."
```

Comment:
Are the space changes necessary? There are several btw.

Reply:
Yeah French rule that i forgot. So they are necessary.
They were like that before.